fe/fe-core/src/main/java/com/starrocks/server/MetadataMgr.java

```diff
@@ -440,6 +440,7 @@ public void dropTemporaryTable(DropTemporaryTableStmt stmt) {
                 if (stmt.isSetIfExists()) {
                     LOG.info("drop temporary table[{}.{}] which doesn't exist in session[{}]",
                             dbName, tableName, sessionId);
+                    return;
```

Comment:
I don't think it is necessary to print this LOG in INFO level.

Reply:
fixed